src/lib/ngsi/Metadata.cpp

```diff
@@ -174,3 +174,46 @@ void Metadata::fill(const struct Metadata& md)
   value       = md.value;
   association = md.association;
 }
+
+
+
+/* ****************************************************************************
+*
+* renderV2 - 
+*/
+std::string Metadata::renderV2(ConnectionInfo* ciP, RequestType request, bool comma)
```

Comment:
Are ciP and request actually used? In negative case, I'd suggest to remove them, in order to keep the function simpler. Probably ciP and request are a leftover of v1 way of working (the XML compataibility probably need them) and we can get rid of it for v2.